File: Pandacap.LowLevel/Data/TheirContent/ActivityPub/PostActivity.fs
namespace Pandacap.Data

open System

/// Another ActivityPub actor's interaction with a Pandacap post.
type PostActivity() =
    member val Id = "" with get, set
    member val InReplyTo = "" with get, set
    member val ActorId = "" with get, set
    member val ActivityType = "" with get, set
    member val AddedAt = DateTimeOffset.MinValue with get, set

File: Pandacap.Database/PostActivity.cs
namespace Pandacap.Database
{
    public class PostActivity
    {
        public string Id { get; set; } = "";
        public string InReplyTo { get; set; } = "";
        public string ActorId { get; set; } = "";
        public string ActivityType { get; set; } = "";
        public DateTimeOffset AddedAt { get; set; }
    }
}
